src/components/dataVisualizations/columnHelpers/playerMatchupsColumnHelper.tsx

```diff
@@ -29,7 +29,7 @@ export const playerMatchupsColumns: ColumnDef<PlayerMatchupItem, any>[] = [
         cell: (info) =>
             info.row.original.matchCount > 0
                 ? `${Math.round((info.row.original.winCount / info.row.original.matchCount) * 100)}%`
-                : `0%`,
+                : `N/A`,
```

Comment:
same comment here that I made in commanderMatchupssColumnHelper.ts

Reply:
same. changed to requested "never happens" comment for now.